src/test/java/com/cognifide/aemrules/htl/HtlProfileTest.java

```diff
@@ -51,7 +51,7 @@ public void sanity() {
         profile.define(context);
         Map<String, BuiltInActiveRule> activeRules = getActiveRulesByRuleKey(context);
 
-        assertThat(activeRules.size()).isEqualTo(4);
+        assertThat(activeRules.size()).isEqualTo(5);
```

Comment:
please update this branch to the master before merge and update those values

Reply:
It is synced with master, and at the moment this test is passing with '5'.